js/app.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+(function(){
+  angular
+  .module("radioapp", [
+    "ui.router",
+    "songsmodule"
+  ])
+  .config([
+    "$stateProvider",
+    RouterFuntion
+  ]);
+
+  function RouterFuntion($stateProvider){
+    $stateProvider
+    .state("songIndex", {
+      url: "/songs",
+      templateUrl: "js/songssection/index.html",
+      controller: "SongsIndexCtrl",
+      controllerAs:"SongsIndexViewModel"
+    })
+    .state("songNew", {
+      url: "/songs/new",
+      templateUrl: "js/songssection/new.html",
+      controller: "SongNewController",
```

Comment:
Be consistent with your controller naming - it will help you maintain your app. Some of your controller names are spelled with `Ctrl` while others use `Controller`.